lib/taiko.js

```diff
@@ -31,6 +31,8 @@ let chromeProcess, temporaryUserDataDir, page, network, runtime, input, client,
 
 let plugins = new Map();
 
+module.exports.emitter = xhrEvent;
```

Comment:
I feel we can use different emitter for descriptions, so that user wont accidentally change the behaviour of other events